raw-mm.valid.main.lua

```diff
@@ -5272,8 +5297,13 @@ function valid.lostwaferbalance()
 end
 
 function valid.notarget()
-  checkaction(dict.checkparalysis.misc, true)
-  lifevision.add(actions.checkparalysis_misc.p, "onclear")
+  if actions.checkparalysis_misc then
+    lifevision.add(actions.checkparalysis_misc.p, "onclear")
+  end
+
+  if actions.healhealth_sip and conf.medicinebag and me.activeskills.stag then
+    config.set("medicinebag",false,true)
```

Comment:
Should set this inside dict instead - that way if lifevision catches a fake, it won't turn it off. I'd show a message to the user here as well to improve chances of them turning the feature back on after the problem is resolved.